src/mappings/irohaMigrationHandler.ts

```diff
@@ -1,35 +1,35 @@
-import {SubstrateExtrinsic} from '@subql/types';
-import {HistoryElement} from "../types";
-import {checkIfExtrinsicExecuteSuccess, formatU128ToBalance, getExtrinsicNetworkFee} from "./utils";
+import { SubstrateExtrinsic } from '@subql/types';
+import { formatU128ToBalance, assignCommonHistoryElemInfo } from "./utils";
 
 export async function handlerIrohaMigration(extrinsic: SubstrateExtrinsic): Promise<void> {
-    
+
     logger.debug("Caught iroha migration extrinsic")
-    
-    const record = new HistoryElement(extrinsic.extrinsic.hash.toString())
 
-    record.blockHeight = extrinsic.block.block.header.number.toBigInt()
+    const record = assignCommonHistoryElemInfo(extrinsic)
+
     record.module = "irohaMigration"
```

Comment:
these fields might be filled in `assignCommonHistoryElemInfo` too.
You can use methods: 
`extrinsic.extrinsic.method.method`
`extrinsic.extrinsic.method.section`